scripts/validate-token-configs.ts

```diff
@@ -0,0 +1,391 @@
+import { ethers } from "ethers";
+import axios from "axios";
+
+/*
```

Comment:
each of these types, constants, and ABIs can be externalized to a different directory